docs/content.zh/getting-started/training-course.md

```diff
@@ -1,5 +1,5 @@
 ---
-weight: 6
```

Comment:
Is this modified by mistake?

Reply:
@PatrickRen  **Weight** will effect sort. I insert before this catalog ,so modify it's weight.

Reply:
Ah thanks for the explanation. I was thinking it was related to font lol.